### DISCUSSION TITLE
Use google charts

### DISCUSSION THREAD

**Discussion by gudge25** [gotenberg/gotenberg#985] [Q&A]
cant fing inf  is it possible to use google charts 
tried to use  by including in HTML  but Got prob ignore it
```
<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
    <script type="text/javascript">
        google.charts.load('current', {'packages':['corechart']});
        google.charts.setOnLoadCallback(drawCharts);
        .....
```

**Comment by gulien** [comment]
Hello @gudge25,

See https://gotenberg.dev/docs/routes#wait-before-rendering-chromium.

Alternatively, you can set the form field `skipNetworkIdleEvent` to `false` if you’re using Gotenberg 8.11. 

First option will work for sure, I don’t know about the second one.
